src/server/scripts/Pandaria/GateoftheSettingSun/boss_commander_rimok.cpp

```diff
@@ -0,0 +1,389 @@
+
+                        events.ScheduleEvent(EVENT_FRENZIED_ASSAULT, urand(10000, 15000));
+                        break;
+                    case EVENT_VISCOUS_FLUID:
+                        me->CastSpell(me, 107077, true);
+                        events.ScheduleEvent(EVENT_VISCOUS_FLUID, urand(5000, 10000));
+                        break;
+                }
+
+                DoMeleeAttackIfReady();
+            }
+
+            void JustDied(Unit* /*killer*/)
+            {
+                _JustDied();
+                Talk(TALK_DEATH);
+            }
+        };
+
+        CreatureAI* GetInstanceAI(Creature* creature) const
+        {
+            return new boss_commander_rimokAI(creature);
+        }
+};
+
+class npc_krikthik_swarmer : public CreatureScript
+{
+    public:
+        npc_krikthik_swarmer() : CreatureScript("npc_krikthik_swarmer") {}
+
+        struct npc_krikthik_swarmerAI : public ScriptedAI
+        {
+            npc_krikthik_swarmerAI(Creature* creature) : ScriptedAI(creature) {}
+
+            uint32 attackTimer;
+
+            void Reset()
+            {
+                attackTimer = 2000;
+            }
+
+            void UpdateAI(uint32 diff)
+            {
+                if (attackTimer)
+                {
+                    if (attackTimer <= diff)
+                    {
+                        DoZoneInCombat();
+                        if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM))
+                            AttackStart(target);
+
+                        attackTimer = 0;
+                    }
+                    else attackTimer -= diff;
+                }
+
+                DoMeleeAttackIfReady();
+            }
+        };
+
+        CreatureAI* GetAI(Creature* creature) const
+        {
+            return new npc_krikthik_swarmerAI(creature);
+        }
+};
+
+class npc_krikthik_saboteur : public CreatureScript
+{
+    public:
+        npc_krikthik_saboteur() : CreatureScript("npc_krikthik_saboteur") {}
+
+        struct npc_krikthik_saboteurAI : public ScriptedAI
+        {
+            npc_krikthik_saboteurAI(Creature* creature) : ScriptedAI(creature) {}
+
+            uint32 attackTimer;
+            uint32 checkTimer;
+
+            void Reset()
+            {
+                attackTimer = 2000;
+                checkTimer = urand(17500, 22500);
+            }
+
+            void UpdateAI(uint32 diff)
+            {
+                if (attackTimer)
+                {
+                    if (attackTimer <= diff)
+                    {
+                        DoZoneInCombat();
+                        if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM))
+                            AttackStart(target);
+
+                        attackTimer = 0;
+                    }
+                    else attackTimer -= diff;
+                }
+
+                if (checkTimer <= diff)
+                {
+                    me->CastSpell(me, SPELL_BOMBARD, false);

```

Comment:
false is default value, so you can remove...
like:  me->CastSpell(me, SPELL_BOMBARD);